examples/main.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   required_providers {
     maas = {
-      source = "registry.terraform.io/ionutbalutoiu/maas"
+      source = "registry.terraform.io/maas/maas"
```

Comment:
@ehaselwanter yes exactly, these changes will go into the master branch just before publishing the provider to the registry, so it will be a valid source.

Reply:
ok. thx. any ETA for that? I guess this starts some sort of "officially" working on it?

Reply:
There's no specific release date determined, but we're aiming for sometime in October/November.